protocol/dubbo/dubbo_invoker.go

```diff
@@ -73,6 +73,9 @@ func NewDubboInvoker(url common.URL, client *remoting.ExchangeClient) *DubboInvo
 	if t, err := time.ParseDuration(requestTimeoutStr); err == nil {
```

Comment:
I think the code should be 

```go
	if t, err := time.ParseDuration(requestTimeoutStr); err == nil {
		requestTimeout = time.Second * 3
	}
```

Reply:
it is because  `time.ParseDuration(requestTimeoutStr); err ==nil` is false, so requestTimeout is 0
I think it not solve the problem